rwgame/main.cpp

```diff
@@ -1,11 +1,10 @@
 #include <iostream>
 #include "RWGame.hpp"
 #include "SDL.h"
-
 #include <core/Logger.hpp>
 
 int main(int argc, char* argv[]) {
-    // Initialise Logging before anything else happens
+    // Initialize Logging before anything else happens
```

Comment:
Oops. Sorry!

I didn't notice the difference between British and American vocabulary at the moment of committing.
My IDE just noticed a spelling error.
Does openrw has a policy on this?

I will happily withdraw this commit.